### PR TITLE
2012 rm slowdown module

### LINKED ISSUE #2012: Rm embox.kernel.time.slowdown module
embox.kernel.time.slowdown module is weird. It's used to make slower system clock, but why system_freq is not been able to use for the aim. Why there some code with this logic exists in clock_tick_handler

```
	if (++cs->jiffies_cnt == (1 << SLOWDOWN_SHIFT)) {
		cs->jiffies_cnt = 0;
		cs->jiffies++;
```
```
templates/x86/test/packetdrill/mods.config:44:	include embox.kernel.time.slowdown(shift=3) /* 1 << 3 */
templates/arm/qt-fb-small/mods.config:39:	include embox.kernel.time.slowdown(shift=2)
templates/arm/qt-app/mods.config:45:	include embox.kernel.time.slowdown(shift=2)
```

### PR DESCRIPTION
**Pull request by anton-bondarev**
Close #2012

### CODE REVIEW

File: src/kernel/time/clock_tick.c [hunk 1]
```diff
@@ -27,15 +25,12 @@ void clock_tick_handler(int irq_num, void *dev_id) {
...
-		cs->jiffies_cnt = 0;
-		cs->jiffies++;
-
-		if (cs_jiffies->event_device &&	irq_num == cs_jiffies->event_device->irq_nr) {
-			if (timer_strat_need_sched(cs_jiffies->jiffies)) {
-				lthread_launch(&clock_handler_lt);
-			}
-		}
+
+	cs->jiffies_cnt = 0;
```
Comment (alexkalmuk) [inline src/kernel/time/clock_tick.c:29]: `jiffies_cnt` seems to be not required anymore

File: src/kernel/time/clock_tick.c [hunk 2]
```diff
@@ -27,15 +25,12 @@ void clock_tick_handler(int irq_num, void *dev_id) {
...
+	cs->jiffies++;
+
+	if (timer_strat_need_sched(cs->jiffies)) {
```
Comment (alexkalmuk) [inline src/kernel/time/clock_tick.c:32]: Also, I proposed here to change `cs` -> `cs_jiffies` to make this code looks more clear.

Comment (alexkalmuk) [reply src/kernel/time/clock_tick.c:32]: Because we are using cs_jiffies in lthread below.

Comment (alexkalmuk) [reply src/kernel/time/clock_tick.c:32]: Or alternatevely you can set `assert (cs == cs_jiffies)` instead.

File: src/kernel/time/clock_tick.c (hunk 1 above)
Comment (anton-bondarev) [reply src/kernel/time/clock_tick.c]: thanks. done

File: src/kernel/time/clock_tick.c (hunk 2 above)
Comment (anton-bondarev) [reply src/kernel/time/clock_tick.c]: agree. done

**Pull request merged**